src/tasks/MonoTargetsTasks/ILStrip/AssemblyStripper/AssemblyStripper.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Why do we need a separate project for the AssemblyStripper if it's just a library and can't be part of the MonoTargetsTasks.csproj?

Reply:
@ViktorHofer because it pulls in the CilStripper sources which are not actively maintained, full of warnings and aren't nullability friendly.  Putting everything in one assembly means we don't get nice checking for all the other tasks.

Reply:
OK. AFAIK the old sources will be replaced anyway with a new tooling during NET7.

Reply:
> OK. AFAIK the old sources will be replaced anyway with a new tooling during NET7.

That's my understanding, too.  At that point this extra assembly can go away